index.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
```

Comment:
nowadays we use "echo framework" instead of Gorilla Mux
please change your framework to default company framework